src/fides/api/models/privacy_experience.py

```diff
@@ -520,3 +672,28 @@ def region_country(region: str) -> str:
     based on naming convention (i.e. `country_subregion`)
     """
     return region.split("_")[0]
+
+
+def link_notices_to_experience_config(
+    db: Session,
+    notice_keys: List[str],
+    experience_config: PrivacyExperienceConfig,
+) -> Tuple[List[PrivacyNoticeRegion], List[PrivacyNoticeRegion]]:
```

Comment:
This helper method lets us add Privacy Notices to a Privacy Experience Config (and unlinks notices not supplied)